src/browser-installer/chrome/index.ts

```diff
@@ -6,41 +6,30 @@ import { DRIVER_WAIT_TIMEOUT } from "../constants";
 import { getMilestone } from "../utils";
 import { installChrome } from "./browser";
 import { installChromeDriver } from "./driver";
-import { isUbuntu, getUbuntuLinkerEnv, installUbuntuPackageDependencies } from "../ubuntu-packages";
+import { isUbuntu, getUbuntuLinkerEnv } from "../ubuntu-packages";
 
 export { installChrome, installChromeDriver };
 
 export const runChromeDriver = async (
     chromeVersion: string,
     { debug = false } = {},
 ): Promise<{ gridUrl: string; process: ChildProcess; port: number }> => {
-    const shouldInstallUbuntuPackageDependencies = await isUbuntu();
-
-    const [chromeDriverPath] = await Promise.all([
+    const [chromeDriverPath, randomPort, chromeDriverEnv] = await Promise.all([
         installChromeDriver(chromeVersion),
-        installChrome(chromeVersion),
-        shouldInstallUbuntuPackageDependencies ? installUbuntuPackageDependencies() : null,
+        getPort(),
+        isUbuntu()
+            .then(isUbuntu => (isUbuntu ? getUbuntuLinkerEnv() : null))
+            .then(extraEnv => (extraEnv ? { ...process.env, ...extraEnv } : process.env)),
     ]);
```

Comment:
Now we dont install browser and ubuntu packages here, and instead we assume it is already installed by "browser-installer/run.ts", which runs "browser-installer/install.ts" before calling "run*Driver"